app/resources/admin.html.template

```diff
@@ -36,6 +36,12 @@
   <input type=submit value="Delete (with confirmation)">
 </form>
 
+<h2>Setup datastore</h2>
+<form method="post" class="admin">
```

Comment:
I did this way because I thought only admins were allowed to perform this step.
However, to get things clearer, what do you mean by "all pages"? When database is not initialized yet, I can only see this:

![selection_010](https://cloud.githubusercontent.com/assets/2883614/10485321/cdca4be8-7289-11e5-851e-a5c9f3cc6bc0.png)

If this is the normal view, I could add the button/link in the sidebar near home. Would be that ok?